pkg/tests/ossm/bug_istiopods_test.go

```diff
@@ -15,79 +15,49 @@
 package ossm
 
 import (
-	"fmt"
-	"math/rand"
-	"strings"
 	"testing"
-	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/examples"
-	"github.com/maistra/maistra-test-tool/pkg/util"
-	"github.com/maistra/maistra-test-tool/pkg/util/env"
-	"github.com/maistra/maistra-test-tool/pkg/util/log"
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-// TestIstioPodProbesFails tests that Istio pod get stuck with probes failure after restart. Jira ticket: https://issues.redhat.com/browse/OSSM-2434
+// TestIstioPodProbesFails tests that Istio pod get stuck with probes failure after restart. Jira ticket bug: https://issues.redhat.com/browse/OSSM-2434
 func TestIstioPodProbesFails(t *testing.T) {
-	test.NewTest(t).Id("T35").Groups(test.Full).NotRefactoredYet()
+	NewTest(t).Id("T35").Groups(Full).Run(func(t TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+		ns := "bookinfo"
+		data := map[string]interface{}{
+			"Count":     50,
+			"Namespace": meshNamespace,
+		}
 
-	const numberOfNamespaces = 50
-	defer func() {
-		log.Log.Info("Cleanup ...")
-		bookinfo := examples.Bookinfo{Namespace: "bookinfo"}
-		bookinfo.Uninstall()
-		util.Shell(fmt.Sprintf(env.GetRootDir()+"/scripts/smmr/clean_members.sh %d", numberOfNamespaces))
-	}()
-	defer util.RecoverPanic(t)
+		t.Cleanup(func() {
+			oc.DeleteNamespaces(t, multiple_namespaces, data)
+			oc.DeleteNamespace(t, ns)
+			oc.RecreateNamespace(t, meshNamespace)
+			oc.ApplyString(t, meshNamespace, smmr)
+		})
 
-	log.Log.Info("Deploy bookinfo in bookinfo ns")
-	bookinfo := examples.Bookinfo{Namespace: "bookinfo"}
-	bookinfo.Install(false)
+		t.LogStep("Install Bookinfo application")
+		app.InstallAndWaitReady(t, app.Bookinfo(ns))
```

Comment:
Why does this test need Bookinfo?

Reply:
It's not needed. IDK why we were installing but in the original Jira regarding the test case we were doing that. Does not have sense, deleted